app/scripts/stores/navigatorStore.js

```diff
@@ -215,16 +217,29 @@ export class NavigatorStore {
         if(item.kind !== Kind.DDS_PROJECT) mainStore.entityObj = item;
     }
     
-    @action setProjPermissions(currentProject) {
-        let projPermissionsCoversion = {
-            'Project Admin': 'prjCrud',
-            'System Admin': 'prjCrud',
-            'Project Viewer': 'viewOnly',
-            'File Editor': 'flCrud',
-            'File Uploader': 'flUpload',
-            'File Downloader': 'flDownload'
+    @action setProjectRole(currentProject) {
+        let projRolesCoversion = {
+            'Project Admin': 'project_admin',
```

Comment:
There's an enum for this.

`import { Roles } from '../../enum';`

It has a slightly different format though. It uses the auth_role id, not the name. 

```javascript
export const Roles = Object.freeze({
    project_admin : 'project_admin',
    system_admin : 'system_admin',
    project_viewer : 'project_viewer',
    file_uploader : 'file_uploader',
    file_downloader : 'file_downloader',
    file_editor : 'file_editor',
});
```

`GET/projects/{project_id}/permissions/{user_id}` returns this format:

```javascript
{
  "project": {
    "id": "ca29f7df-33ca-46dd-a015-92c46fdb6fd1"
  },
  "user": {
    "id": "c1179f73-0558-4f96-afc7-9d251e65b7bb",
    "username": "mrgardner01",
    "full_name": "Matthew Gardner"
  },
  "auth_role": {
    "id": "file_editor",
    "name": "File Editor",
    "description": "Can view, download, create, update and delete files"
  }
}
```